### ISSUE TITLE
Watchlist backtest

### ISSUE THREAD

**Issue by Ham43615** [akumidv/tradingview-assistant-chrome-extension#92]
Ability to backtest watchlist with timeframe

**Comment by Buddingm** [comment]
I asked it a long time ago but he did not implement it but said to do it.

**Comment by akumidv** [comment]
It will increase number of testing and increase risk of bun from TV.  Are you sure that this functionality is helpful?

**Comment by Buddingm** [comment]
I'm 1000% sure it's usefull. I use it each day with David's version. You not get banned when running only 1 tab. I use it almost 24/7

**Comment by Camilo-Mora** [comment]
this app, is already quite amazingly helpful and I concur this functionality will be useful. It will certainly save hour to my team.

**Comment by Buddingm** [comment]
Camilo, you can use the chrome Extension from the chrome Store. That is currently working.

**Comment by Camilo-Mora** [comment]
> Camilo, you can use the chrome Extension from the chrome Store. That is currently working.

You mean the function to run the back testing on the watch list is on the Chrome Store? I downloaded it and did not see it... only the David one, has it.

**Comment by Camilo-Mora** [comment]
> Yes. David's version is in the chrome store and much more advanced with for example watch list

Yes, I am aware of that option in David's version. The problem is that they removed the "Brutal force" option from their optimization, which I like the most. Also they have an internal line of  code that send internal information of your computer to their discord channel...that I did not like.

**Comment by Buddingm** [comment]
You can disable the option to sent to discord. I don't know what brutal Force did. Maybe the name is changed? There are now 2 options called : random

**Comment by GitHub1m** [comment]
This would be very beneficial!

**Comment by mikeinvest** [comment]
I also suggest adding this feature very helpful.
